src/utils/file.js

```diff
@@ -13,3 +13,7 @@ export function getContent(filename) {
   }
   return promises[filename];
 }
+
+export function setContent(filename, content) {
```

Comment:
This will be used in tests to fake the content instead of writing on disk on temp files.

Temp files will still be used in case there is a `require`.